### PR TITLE
C_CPP: Pre-Defined Manipulator

### LINKED ISSUE #2334: C_CPP: Pre-Defined Manipulators
### Description

documentation

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by 124015001**
<hr>

## Description 📜

Explained about Pre-Defined Manipulators. Also explained different types of Manipulators with their particular programs as an example.
Fixes #2334 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

File: C_CPP/File Handling & IO Management/readme.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 
+- [Pre-Defined Manipulator]("pre-defined-manipulators.md")
 - [Stream Class Model in C++]("stream-class-model.md")
```
Comment (vasu-1) [inline C_CPP/File Handling & IO Management/readme.md:3]:
```suggestion
- [Pre-Defined Manipulator](pre-defined-manipulators.md)
- [Stream Class Model in C++](stream-class-model.md)
```

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Great Job !
looking cool !

just do this slight change
- remove the double quotes from both links

**Review by 124015001 — COMMENTED** [review]
Made the appropriate changes.

**Review by vasu-1 — APPROVED** [review]
I think good to go for merge.
@Suvraneel

File: C_CPP/File Handling & IO Management/pre-defined-manipulators.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    
+    |   |   |   | 4 | 2 |
+    |---|---|---|---|---|
+    | 4 | 2 |   |   |   |
+  </div>
+  
+<br />
+  
+  
+- ## _setpricision(int p)_:
```
Comment (Suvraneel) [inline C_CPP/File Handling & IO Management/pre-defined-manipulators.md:42]:
```suggestion
- ## _setprecision(int p)_:
```

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@124015001 Great documentation ❤️

**Pull request merged**